include/vtzero/builder.hpp

```diff
@@ -228,7 +228,7 @@ namespace vtzero {
         void set_point_impl(const point<Dimensions> p) {
             point<Dimensions> q{sub(p.x, m_cursor.x), sub(p.y, m_cursor.y)};
             if (Dimensions == 3) {
-                q.set_z(sub(p.get_z(), m_cursor.get_z()));
+                q.set_z(p.get_z() - m_cursor.get_z());
```

Comment:
Signed integer overflow is UB. This can overflow because there are arbitrary numbers involved here. The old implementation was carefully done in a way this couldn't happen.